src/components/TableforTopic.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+  <div class="table-responsive">
+    <table id="table-content" class="table">
+      <thead style="width:100%">
+        <tr style="width:100%">
+          <th style="width:7%; border-right:1px solid #D8D8D8;" scope="col">{{tableName}}<img src="../../src/assets/Sorting.svg" v-on:click="sortTableDatabyName" class="sort-button"/></th>
+          <th style="width:3.77%" scope="col">Jan</th>
```

Comment:
Let's have an array with months names as a helper and iterate to avoid duplication of code